### DISCUSSION TITLE
Quotation marks

### DISCUSSION THREAD

**Discussion by sonarventu** [joaomlourenco/novathesis#453] [Q&A]
Hello,
I noted that even by explicitly writing ``something'', the quotation marks are not displayed correctly in the text -- i.e., the opening and closing marks are displayed with the same symbol.

Is there a way to fix this?

Thanks! 
Davide

**Comment by joaomlourenco** [comment]
Can you please post here an example of your source and the produced PDF?

**Reply by sonarventu** [reply]
Sorry, I closed the discussion because I realized that it's not true -- opening and closing marks do not have the same symbol (although they are actually very very similar). You can eliminate this question altogether if you want...
Thanks, Davide
